<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-7R4ES4M" failures="1" tests="3" name="Alumni Module" time="43.543" errors="0" timestamp="2024-08-26T14:23:42 IST">
  <testcase classname="Test_Alumni.SC_AL_001_Test_LoginTest" name="TC001_VerifyLogin" time="12.135"/>
  <testcase classname="Test_Alumni.SC_AL_001_Test_LoginTest" name="TC002_VerifyInavlidEmail" time="8.708">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at Test_Alumni.SC_AL_001_Test_LoginTest.TC002_VerifyInavlidEmail(SC_AL_001_Test_LoginTest.java:41)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 34 stack frames]]>
    </failure>
  </testcase> <!-- TC002_VerifyInavlidEmail -->
  <testcase classname="Test_Alumni.SC_AL_001_Test_LoginTest" name="TC003_VerifyInvalidPassword" time="9.04"/>
</testsuite> <!-- Alumni Module -->
